extensions/2.0/Vendor/EXT_meshopt_compression/README.md

```diff
@@ -118,6 +120,8 @@ While the extension JSON specifies a separate buffer to source compressed data f
 { "byteLength": 1432878 }
 ```
 
+The `byteLength` property of such a dummy buffer **MUST** be sufficiently large to contain all uncompressed bufferViews referencing it.

```

Comment:
Please replace "dummy" with "placeholder" here and in the next paragraph.

Reply:
Thanks for quickly looking at this. While this wording was the original choice of the extension authors (likely based on earlier discussions), let me make this change for better readability.